src/index.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+
+// Create global app object
+const app = express();
+
+// Normal express config defaults
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.get('/', (req, res) => {
```

Comment:
No shorthand. Use `request` and `response`

Reply:
Alright